data/en/contact.yml

```diff
@@ -7,17 +7,14 @@ contact:
   contact_details:
     enable : true
     title: Contact Details
-    subtitle: We love to talk through complex data issues and building with event-driven architecture. Drop us a line for a friendly chat. We’ll make it worth your while.
+    subtitle: We love to talk through resolving complex data challenges with AI driven tools. Drop us a line for a friendly chat. We'll make it worth your while.
```

Comment:
Changing the copy has been discussed, so I expect an additional updates. For now, I wanted to replace the `event-driven` language.